### ISSUE TITLE
[SAP][Button]: On mouse hover button border fails to meet color contrast ratio of 3:1

### ISSUE THREAD

**Issue by rama1990br** [SAP/ui5-webcomponents#6840]
### Describe the bug

On mouse hover button border fails to meet color contrast ratio of 3:1.

### Reproduction steps

**Instance:**
REMOVED - see internal slack channel

**Steps:**
Login to instance.
Directly goto link given
Script-Based Integration Settings page is opened. Goto Add Vendor button
Verify color on mouse hover.
Then on Script-Based Integration Settings, verify color for all button/UI elements on mouse hover

**Actual Results:**
On mouse hover, buttons have light grey border with color ratio of 1.18 which does not meet 3:1 ratio

### Expected Behaviour

The text and border foreground color and background color of each interactive element should have a contrast ratio of at least 3:1. 

### Screenshots or Videos

![info icon border color issue](https://user-images.githubusercontent.com/32254036/229024881-7b58ecd9-9c96-4746-8262-2d15896ba5dd.png)
![border color ratio issue](https://user-images.githubusercontent.com/32254036/229024883-2d6e8138-4c93-4f4f-b1da-4f48e9e20906.png)

### UI5 Web Components for React Version

1.5.1

### UI5 Web Components Version

1.9.1

### Browser

Chrome, Edge, Firefox, Safari

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by unazko** [comment]
Hello @SAP/ui5-webcomponents-topic-b,

This issue is reproducible via the following playground same for the "Default" and "Transparent" design types:
https://sap.github.io/ui5-webcomponents/playground/components/Button/

Graphical components should have contrast ratio of 3:1 between one another.

For those design types this ratio is 1.5:1.
We would need to check this one with the visual design colleagues.

Best regards,
Boyan

**Comment by rama1990br** [comment]
Hi Team,
Can we please have an update on this ticket?
